### DISCUSSION TITLE
Deploy in K8s

### DISCUSSION THREAD

**Discussion by jg0rd042** [reconmap/reconmap#85] [Ideas]
It could be interesting to be able to deploy it in a kubernetes cluster, to use scalability for cases of large pentesting teams.

**Comment by santiagolizardo** [comment]
I have updated the [Reconmap Kubernetes repository](https://github.com/reconmap/k8s-deployment) with the latest version of the containers. It can't be ran without adjusting configuration values (look at the configmap files) but it should be a great starting point to whoever wants to run it on a K8s cluster.
Let me know if that's working well with you. Thanks!
